web/scripts/retrieve_data.py

```diff
@@ -90,6 +95,73 @@
 A2_MIDDLE_EAST = [pycountry_convert.country_name_to_country_alpha2(c).lower() for c in _middle_east]
 assert not any([c is None for c in A2_MIDDLE_EAST]), f"Null country in {A2_MIDDLE_EAST}"
 
+GROUP_ID_TO_NAME = {
+    "sp500": "S&P 500",
+    "globalBigTech": "Global Big Tech",
+    "genAI": "GenAI Contenders"
+}
+
+CORE_COLUMN_MAPPING = OrderedDict([
```

Comment:
This mapping does not include the following columns defined in the PARAT dataset docs:
* PARAT link
* City
* State/province
* Parent company ID

Are any of these needed?